viite-UI/src/view/NodeForm.js

```diff
@@ -140,63 +140,51 @@
     };
 
     var Junctions = function () {
-      var headers = function(asResume) {
-        var checkbox = '';
-        if (!asResume) {
-          checkbox += '<th class="node-junctions-table-header">' +
-            ' <table class="node-junctions-table-dimension">' +
-            '   <tr><th class="node-junctions-table-header">Irrota</th></tr>' +
-            '   <tr><th class="node-junctions-table-header">liittymä</th></tr>' +
-            '   <tr><th class="node-junctions-table-header">solmusta</th></tr>' +
-            ' </table>' +
-            '</th>';
-        }
+      var headers = function(options) {
         return '<tr class="node-junctions-table-header-border-bottom">' +
-          checkbox +
-          '<th class="node-junctions-table-header">NRO</th>' +
-          '<th class="node-junctions-table-header">TIE</th>' +
-          '<th class="node-junctions-table-header">AJR</th>' +
-          '<th class="node-junctions-table-header">OSA</th>' +
-          '<th class="node-junctions-table-header">ET</th>' +
-          '<th class="node-junctions-table-header">EJ</th>' +
+          ((options && options.checkbox) ? '<th class="node-junctions-table-header">' +
```

Comment:
You might use backtick Template literals (Template strings) here instead of quote character. It works similar as scala s""" It is supported by chrome and Firefox and not by IE. We are not supporting IE so it can be used safely.